dls-controls/malcolmjs
# Feature/testing strategy #194

## Description

Added a testing strategy section as well as a skeleton for the git workflow.

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- build docs and read

## Agile board tracking

connect to #194 
connect to #186 

closes #194
closes #186
